resources/chef_ingredient.rb

```diff
@@ -52,14 +52,17 @@
 platform_family = node['platform_family']
 
 action_class do
+  require_relative '../libraries/helpers'
```

Comment:
Not according to our custom resource docs. All modules in libraries are auto-loaded. Is there a style guide you are referring to?